android/gointegration/build.GHAgradle

```diff
@@ -31,20 +32,44 @@ android {
     }
 
     repositories {
-        flatDir{
-            dirs 'src/libs'
+        maven {
```

Comment:
build.GHAgradle should use the generated .aar from the Go code binding in the CI/CD flow instead of the version from maeven